### ISSUE TITLE
"Type application/netcdf is already registered as a variant of application/netcdf." when using Zammad console

### ISSUE THREAD

**Issue by JsBergbau** [zammad/zammad#4038]
### Infos:

* Used Zammad version: 5.4.x 
* Installation method (source, package, ..): Package like described there https://docs.zammad.org/en/latest/install/package.html
* Operating system:  Debian GNU/Linux 10 (buster)
* Database + version: not relevant
* Elasticsearch version: not relevant
* Browser + version: not relevant

Every minute there is a cronjob to use Zammad privacy friendly and runs `zammad run rails r "ActivityStream.destroy_all; OnlineNotification.destroy_all"`

### Expected behavior:

* No error message

### Actual behavior:

* A few times a day an E-Mail containing `Type application/netcdf is already registered as a variant of application/netcdf.` is sent. So this problem occurs only sometimes.

### Steps to reproduce the behavior:

* Create a cronjob like above every minute.

This error occurs in the latest Zammad version 5.1.x up to 5.4.x. With 5.0.x this error never occured. 

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
I've seen this once or twice on environments as well. Didn't see it in like a month any more and also cannot reproduce it so that an developer could fix it.

Please also note that your cronjob approach is potentially an isuse. If you have questions about this consult the community: https://community.zammad.org

**Comment by JsBergbau** [comment]
Did you try to run this Cronjob every minute? Sometimes it happens a few times a day, then there are days without any occurence. Can you explain how the cronjob could be the issue? Basically it just runs the zammad shell. Doing this via cron or not shouldn't make any difference, should it?

**Comment by martini** [comment]
Update: I did have similar issue (sometimes). After some research it seems that it is fixed with ruby gem `mime-types-data` version 3.2023.0220 / 2023-02-18, see also https://github.com/mime-types/mime-types-data/blob/main/History.md#320230218--2023-02-18, https://github.com/mime-types/mime-types-data/pull/50 and https://github.com/mime-types/ruby-mime-types/issues/163

```
This release also fixes [ruby-mime-types#163][], where logs show "Type
  application/netcdf is already registered as a variant of
  application/netcdf".
```

**Comment by martini** [comment]
Can get reproduced within 300 times by using `while true ; do rails r 'p Setting.get("fqdn")' ; date ; done`

With `mime-types-data` version 3.2023.0220 / 2023-02-18 it`s gone.
